examples/lifelong_learning/atcii/README.md

```diff
@@ -16,15 +16,15 @@ In this example, you can use [ASHRAE Global Thermal Comfort Database II](https:/
 
 
 
-download [datasets](https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/atcii-classifier/dataset.tar.gz), including train, evaluation and incremental dataset.
+We provide a well-processed [datasets](https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/atcii-classifier/dataset.tar.gz), which including train, evaluation and incremental dataset.
 ```
-cd /
+cd /data
 wget https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/atcii-classifier/dataset.tar.gz
-tar -zxvf dataset.tar.gz
+tar -zxvf dataset.tar.gz  # Note: In this package, testData.csv is the evaluation dataset, trainData.csv is the train dataset and trainData2 will be the incremental dataset
```

Comment:
Better to append this note into `We provide a well-processed [datasets]` section instead of just embeding into the commands

Reply:
fix